multi-cluster-service/aws/setup-aws.sh

```diff
@@ -0,0 +1,65 @@
+export CLUSTER_MAIN=${1:-"psmdb-cluster-mcs-setup-test-1"}
+export CLUSTER_REPL=${2:-"psmdb-cluster-mcs-setup-test-2"}
+export CLUSTER_MAIN_CIDR=10.10.0.0/16
+export CLUSTER_REPL_CIDR=10.12.0.0/16
+
```

Comment:
maybe check if eksctl, aws, kubectl and gcloud binaries exist, exit if not (just a thought to prevent executing half commands if some exist and some don't)

Reply:
Might we just add some info about requirments?